ci-operator/config/openshift/oauth-proxy/openshift-oauth-proxy-master.yaml

```diff
@@ -17,9 +17,6 @@ images:
 - dockerfile_path: Dockerfile.rhel7
   from: base
   inputs:
-    bin:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```

Comment:
@stlaz pls fix the name in the dockerfile https://github.com/openshift/oauth-proxy/blame/master/Dockerfile.rhel7#L1